src/main/java/cn/nukkit/block/BlockTallGrass.java

```diff
@@ -123,8 +125,8 @@ public boolean onActivate(@Nonnull Item item, Player player) {
 
     @Override
     public Item[] getDrops(Item item) {
-        boolean dropSeeds = ThreadLocalRandom.current().nextInt(10) == 0;
         if (item.isShears()) {
```

Comment:
I think we should use a `new ArrayList(2)` to compute the drops instead and add the `tall_grass` item as a guaranteed drop if using a shear, and then let the rest of the code check the enchantment and decide if the seeds should be added to the drop list, at the end of the method convert the list to an array with `return drops.toArray(new Item[0]);`
This way the logic would also work if the shear is enchanted with fortune